docs/css/home.css

```
@@ -4,6 +4,9 @@
   display: flex;
   flex-wrap: wrap;
   justify-content: space-between;
+}
+
+.cards-box.spacing {
```

Maybe we could use the BEM convention and name it something like a `spaced` modifier.
```suggestion
.cards-box--spaced {
```
What do you think?

---

909e8bf